code/__HELPERS/_logging.dm

```diff
@@ -134,6 +134,10 @@
 	if (CONFIG_GET(flag/log_emote))
 		WRITE_LOG(GLOB.world_game_log, "EMOTE: [text]")
 
+/proc/log_subtler(text)
+	if (CONFIG_GET(flag/log_subtler))
+		WRITE_LOG(GLOB.world_game_log, "SUBTLER EMOTE: [text]")
+
```

Comment:
Could you move this to somewhere inside "code\__HELPERS\~skyrat_helpers"? Be it in unsorted or you could make a "logging.dm"

Reply:
Also done.